### PR TITLE
Viite 926, 918 drawing order in renumbering+flipping direction

### PR DESCRIPTION
**Pull request by oappicgi**

### CODE REVIEW

File: digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/ProjectService.scala [hunk 1]
```diff
@@ -1022,6 +1022,7 @@ class ProjectService(roadAddressService: RoadAddressService, roadLinkService: Ro
             ProjectDAO.getProjectLinksByLinkId(updatedProjectLinks.head.linkId).headOption match {
               case Some(roadPartLink) =>
                 if (roadPartLink.roadNumber == newRoadNumber && roadPartLink.roadPartNumber == newRoadPartNumber)
+                  if (ProjectDAO.getProjectLinksByProjectAndLinkId(linkIds,projectId).exists(x=>x.status!=LinkStatus.Numbering)) //check if renumbering has already been done (and we are actually doing ie direction change)
```
Comment (tvenhola) [inline digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/ProjectService.scala]: Should we instead check that the renumbering is not trying to use the same number as the referenced road address object?

Comment (oappicgi) [reply digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/ProjectService.scala]: that would be cleaner way

Comment (oappicgi) [reply digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/ProjectService.scala]: i will change the logic to that

**Comment by oappicgi** [comment]
didnt have change to run test on latest commit

**Pull request merged**